tests/connection_e2e_fcr2aws_test.go

```diff
@@ -0,0 +1,21 @@
+package tests
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCloudRouter2AwsCreateConnection(t *testing.T) {
+	// retryable errors in terraform testing.
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		TerraformDir: "../examples/fabric-cloud-router/fg2aws",
+	})
+
+	defer terraform.Destroy(t, terraformOptions)
+
+	terraform.InitAndApply(t, terraformOptions)
+	output := terraform.Output(t, terraformOptions, "fg2aws_connection_result")
```

Comment:
Output needs to match the output name in the main.tf of the terraform it is referencing. In this case the output is "connection_result"

Reply:
Ack.